sig-security/sig-security-tooling/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
is there an additional subproject owner it would make sense to add for redundancy? xref guidance in https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#subproject-owner

Reply:
Not at this time; as the subprojects grow into their maturity we will add backups; for now, the sig chairs are backing up the subproject owners.

Reply:
would it be worth adding the sig chairs here to make it clear there's a plurality?

Reply:
@liggitt just discussed this with @IanColdwater in our sig meeting. Are you suggesting in addition to myself adding both chairs as approvers and reviewers or just as approvers?

Reply:
multiple approvers is the main goal. inclusion in reviewers is up to you all, depending on what you want... that primarily just informs the bot auto-assignments

Reply:
Done.